string command # takeoff, RTH, land, sweep, center
---
bool success
